src/psyclone/tests/parse/kernel_test.py

```diff
@@ -43,14 +43,23 @@
 import os
 import pytest
 from fparser.api import parse
+from fparser import api as fpapi
```

Comment:
I am afraid this test module only works when the `pytest` is run from the `src/psyclone/tests/parse` directory where the file is located. It should work regardless of the location when running `pytest`.

Reply:
I confirm this error too.

Reply:
Thanks for fixing, all works now from the top working copy directory and from the `src/psyclone/tests` directory.